packages/app-project/src/helpers/GrommetWrapper/GrommetWrapperContainer.js

```diff
@@ -25,7 +25,7 @@ class GrommetWrapperContainer extends Component {
     const mergedThemes = this.mergeThemes()
 
     return (
-      <Grommet theme={mergedThemes}>
+      <Grommet userAgent='Mobile' theme={mergedThemes}>
```

Comment:
What is this `userAgent` prop doing?

Reply:
Setting the default screen size to small, as per the Grommet docs. Without it, it defaults the `screenSize` in `ResponsiveContext` to `medium`, which would jam up on mobile devices with small screens. Also, it'll save an API request on the `RecentTalkSubjects` widget, as it won't get those subjects by default on mobile; it'll adjust once `ResponsiveContext` gets the actual screen size, and then make the request if required.

Reply:
To be fair, we could just trust that user agent detection in Grommet is sensible and request / render based on that

Reply:
Ok, but this now has the opposite problem, where we see a flash of the small screen layout before switching to the desktop layout when on a desktop large size.

The small screen layout also is showing recent subjects in a carousel now, so saving that network request isn't necessary anymore.

Reply:
I might try and implement https://github.com/artsy/fresnel with Grommet's breakpoints then